apps/prairielearn/src/sync/syncFromDisk.js

```diff
@@ -39,12 +39,12 @@ const perf = makePerformance('sync');
  * @param {any} logger
  * @returns Promise<SyncResults>
  */
-export async function syncDiskToSqlWithLock(courseDir, courseId, logger) {
```

Comment:
Nit: can't comment on three lines above, but the order in the comment doesn't match the arguments now.

Reply:
Good catch, fixed in 0e94d34.